### DISCUSSION TITLE
conda instead of venv?

### DISCUSSION THREAD

**Discussion by MSDNAndi** [microsoft/autogen#4250] [General]
Hello together,

wonder if anybody has written up the steps to use conda instead of venv when installing from source?

Kind regards,

**Comment by jackgerrits** [comment]
Conda should work normally. The main difference being that since you're not using `uv` you'll need to install the packages in the right order to satisfy the dependencies.

Within your conda environment:
```sh
pip install -e python/packages/autogen-core
pip install -e python/packages/autogen-ext
pip install -e python/packages/autogen-agentchat
```

Let me know if you face any issues
